### PR TITLE
U/jrbogart/update postgres truth

### PR DESCRIPTION
**Pull request by JoanneBogart**

### CODE REVIEW

**Comment by JoanneBogart** [comment]
Updates Postgres_truth notebook for November Sprint week, in particular add example of use of dr6 object catalog along with truth.

File: tutorials/postgres_forcedsource.ipynb [hunk 1]
```diff
@@ -12,7 +12,7 @@
    "metadata": {},
    "source": [
     "Owner: **Joanne Bogart [@jrbogart](https://github.com/LSSTDESC/DC2-analysis/issues/new?body=@jrbogart)**  \n",
-    "Last Verified to Run: **2020-08-03**\n",
+    "Last Verified to Run: \n",
```
Comment (yymao) [inline tutorials/postgres_forcedsource.ipynb:15]: Add date? 
```suggestion
    "Last Verified to Run: 2020-11-24\n",
```

File: tutorials/postgres_object_1_intro.ipynb [hunk 2]
```diff
@@ -12,7 +12,7 @@
    "metadata": {},
    "source": [
     "Owner: **Joanne Bogart [@jrbogart](https://github.com/LSSTDESC/DC2-analysis/issues/new?body=@jrbogart)**  \n",
-    "Last Verified to Run: **2020-08-03**\n",
+    "Last Verified to Run: \n",
```
Comment (yymao) [inline tutorials/postgres_object_1_intro.ipynb:15]: Add date? 
```suggestion
    "Last Verified to Run: 2020-11-24\n",
```

File: tutorials/postgres_object_1_intro.ipynb [hunk 3]
```diff
@@ -385,8 +385,6 @@
     "## SQL Primer\n",
     "This optional section is intended for those with little or no prior experience with an SQL database, in particular with a server-mediated database like PostgreSQL. Most of the material discussed here applies to SQLite as well but not all; differences are pointed out. Since the PostgreSQL database is read-only for users write operations like `INSERT` and `UPDATE` are not discussed.\n",
     "\n",
-    "**NOTE:** The most up-to-date version of the Primer and following sections may be found [in Confluence](https://confluence.slac.stanford.edu/x/s4joE)\n",
```
Comment (yymao) [inline tutorials/postgres_object_1_intro.ipynb:388]: are you intended to remove this link to the Primer in this notebook?

File: tutorials/postgres_truth.ipynb [hunk 4]
```diff
@@ -12,9 +12,9 @@
    "metadata": {},
    "source": [
     "Owner: **Joanne Bogart [@jrbogart](https://github.com/LSSTDESC/DC2-analysis/issues/new?body=@jrbogart)**  \n",
-    "Last Verified to Run: **2020-08-03**\n",
+    "Last Verified to Run: **2020-11-24**\n",
     "\n",
-    "This notebook demonstrates access to truth catalog data via the PostgreSQL database at NERSC.  Currently (May 1, 2020) two kinds of truth catalogs are available: star truth and supernova truth.  This notebook will concentrate on star truth. The minion observation database is also available \n",
+    "This notebook demonstrates access to truth catalog data via the PostgreSQL database at NERSC.  Currently (May 1, 2020) two kinds of truth catalogs are available: star truth and supernova truth.  This notebook will concentrate on star truth. The minion observation database is also available, as well as the Run2.2i dr6 object catalog (dpdd columns approximating specification in LSE-163 only).\n",
```
Comment (yymao) [inline tutorials/postgres_truth.ipynb:17]: Add a link to LSE-163? Also, note that the version we used is 2018 (I just noticed that LSE-163 has an update in 2020)

**Review by yymao — APPROVED** [review]
Looks good. I add some minor suggestions.

File: tutorials/postgres_object_1_intro.ipynb (hunk 2 above)
Comment (JoanneBogart) [reply tutorials/postgres_object_1_intro.ipynb:15]: I haven't recently rerun these.  The old date of 2020-08-03 would be right -- or I could and probably should rerun.

File: tutorials/postgres_truth.ipynb (hunk 4 above)
Comment (JoanneBogart) [reply tutorials/postgres_truth.ipynb:17]: Good idea!

File: tutorials/postgres_forcedsource.ipynb (hunk 1 above)
Comment (JoanneBogart) [reply tutorials/postgres_forcedsource.ipynb]: Reverted to previous version, which was correct.

File: tutorials/postgres_object_1_intro.ipynb (hunk 2 above)
Comment (JoanneBogart) [reply tutorials/postgres_object_1_intro.ipynb]: Reverted to previous version.

File: tutorials/postgres_object_1_intro.ipynb (hunk 3 above)
Comment (JoanneBogart) [reply tutorials/postgres_object_1_intro.ipynb]: No, I must have committed the wrong version somehow.  Will fix.

Comment (JoanneBogart) [reply tutorials/postgres_object_1_intro.ipynb]: reverted to previous version.

File: tutorials/postgres_truth.ipynb (hunk 4 above)
Comment (JoanneBogart) [reply tutorials/postgres_truth.ipynb]: Link added.

**Pull request merged**